### PR TITLE
fix Issue 15549

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

**Comment by 9il** [comment]
https://github.com/DlangScience/mir/pull/5#issuecomment-170407203

**Comment by JackStouffer** [comment]
Needs a unit test.

**Comment by 9il** [comment]
> Needs a unit test.

Thanks! I forgot to push it.

**Comment by 9il** [comment]
Need some additional work.

**Comment by 9il** [comment]
It is correct :) (Oh.. very abstract code about very abstract things)

File: std/experimental/ndslice/selection.d [hunk 1]
```diff
@@ -1284,6 +1302,21 @@ Use $(SUBREF iteration, allReversed) in pipeline before
     foreach (elem; elems0) {}
 }
 
+/// Issue 15549
```
Comment (JackStouffer) [inline std/experimental/ndslice/selection.d]: Why DDoc?

Comment (9il) [inline std/experimental/ndslice/selection.d]: fixed

**Comment by andralex** [comment]
"I have no idea what I'm doing" :o)

**Comment by 9il** [comment]
Thanks! X))

**Pull request merged**